db/schema.sql

```diff
@@ -5,5 +5,46 @@ CREATE TABLE IF NOT EXISTS events (
     description TEXT,
     start_date TIMESTAMP NOT NULL,
     end_date TIMESTAMP NOT NULL,
+    volunteers_required INTEGER NOT NULL,
     created_at TIMESTAMP NOT NULL DEFAULT NOW(),
-)
\ No newline at end of file
+);
+
+--- Users
+CREATE TABLE IF NOT EXISTS users (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(255) NOT NULL,
```

Comment:
For users, could email be used as a primary key for users? In general, each user should be associated with exactly one email

Reply:
email is unique for users (as specified by schema currently), we can add an index to make queries on it faster. However, having varchar as a foreign key is usually not the best idea.